docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -333,6 +344,22 @@ The produced executable will be a 64 bit Linux executable, so depending on your
 it may no longer be runnable.
 However, it's not an issue as we are going to copy it to a Docker container.
 
+Another way of customizing the native build image build process is to define a custom task. If for example it is required to set the `enableHttpUrlHandler`, it can be done like so:
+
+[source,groovy]
+----
+task customBuildNative(type: io.quarkus.gradle.tasks.QuarkusNative) {
+    enableHttpUrlHandler = true
+}
```

Comment:
You can't configure an existing task?

Reply:
Nope... I tried it and gradle failed. Unless I did it wrong :)

Reply:
Hm, maybe there is some other trick

Reply:
I think I got it to work. I'll update the docs momentarily. Thanks for nudging me in the right direction :P